resumes/views.py

```diff
@@ -57,3 +58,37 @@ def post(self,request):
             return JsonResponse({'MESSAGE':'KEY_ERROR'},status=400)
         except ValueError:
             return JsonResponse({'MESSAGE':'VALUE_ERROR'},status=400)
+
+    @login_required
+    def get(self,request,resume_id=None):
+        user = request.user
+
+        if not resume_id:
+            results = {
+                'resume_title': f"{user.name}_{user.resume_set.count() + 1}",
+                'resume_name': user.name,
+                'resume_email': user.email,
+                'resume_phone_number': user.phonenumber
+            }
+            return JsonResponse({'MESSAGE': 'SUCCESS', 'RESULTS': results}, status=200)
+        if not Resume.objects.filter(id=resume_id).exists():
+            return JsonResponse({'MESSAGE':'NOT_FOUND_RESUME'},status=404)
+        resume  = Resume.objects.get(id=resume_id)
```

Comment:
```suggestion
            }
            return JsonResponse({'MESSAGE': 'SUCCESS', 'RESULTS': results}, status=200)
            
        if not Resume.objects.filter(id=resume_id).exists():
            return JsonResponse({'MESSAGE':'NOT_FOUND_RESUME'},status=404)
        
        resume  = Resume.objects.get(id=resume_id)
```

가독성을 위해 다음과 같이 공백을 추가하면 좋을 것 같습니다.

Reply:
수정했습니다.